core/src/main/java/org/helsinki/paytrail/model/refunds/PaytrailRefundResponse.java

```diff
@@ -1,14 +1,17 @@
 package org.helsinki.paytrail.model.refunds;
 
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.experimental.Accessors;
 import org.helsinki.paytrail.constants.RefundStatus;
+import org.helsinki.paytrail.response.PaytrailResponse;
 
 import java.io.Serializable;
 
 @Data
+@EqualsAndHashCode(callSuper = true)
 @Accessors(chain = true)
-public class PaytrailRefundResponse implements Serializable {
+public class PaytrailRefundResponse extends PaytrailResponse implements Serializable {
```

Comment:
This inheritance is unnecessary, because PaytrailRefundCreateResponse already extends it. All the fields in PaytrailResponse is available from there.

Reply:
Fixed